src/child/index.js

```diff
@@ -25,14 +25,18 @@ require('script!../../node_modules/d3fc/dist/d3fc.bundle.min.js');
 require('script!../../node_modules/BitFlux/dist/bitflux.js');
 /* eslint-enable import/no-unresolved */
 
+const { store, parent } = window.opener;
+
 currentWindowService.ready(() => {
     const currentWindow = currentWindowService.getCurrentWindow();
-    const store = currentWindow.contentWindow.opener.store;
     const rootElement = document.getElementById('app');
 
     const windowStateService = new WindowStateService(currentWindow, store, rootElement);
     windowStateService.start();
 
+    window.store = store;
+    window.parent = parent;
+
```

Comment:
`store` is used here to cover the case when the opener window is closed before the spawned window reaches the ready state. `parent` doesn't seem to be used, must've been left over after a rebase!